dotcom-rendering/src/components/SignInGate/dismissGate.test.ts

```diff
@@ -8,19 +9,14 @@ import {
 
 describe('SignInGate - dismissGate methods', () => {
 	beforeEach(() => {
-		localStorage.clear();
```

Comment:
it feels reasonable to use the native API in tests to me - here you want to get at the native layer i think. there's less benefit in using the helper, and i can well imagine wanting access to native layer if the helpers become dependent on consent?